stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -1369,6 +1369,79 @@ function sparse_sortedlinearindices!(I::Vector{Ti}, V::Vector, m::Int, n::Int) w
     return SparseMatrixCSC(m, n, colptr, I, V)
 end
 
+# findfirst/next/prev/last
+import Base: findfirst, findnext, findprev, findlast
```

Comment:
`findfirst` and `findlast` aren't overloaded here. Also, this file seems to use `Base.f` rather than `import Base: f`.